### ISSUE TITLE
Do we need to run crypt -e example3 to crypt 

### ISSUE THREAD

**Issue by vkkindia** [kutoga/libtbsbincr#2]
After compiling the program, is the example3 itself a encrypted one , or do we need to run crypt tool to encrypt.
Because when i run 
' ' ' ' 
./bin/tbs/bincr/crypt  -d example3_no_crypt 
Decrypted 0 sections for 'example3_no_crypt'
' ' ' ' 
./bin/tbs/bincr/crypt  -e example3 
Use the seed 2084265814 for the encryption.
Encrypted 6 sections for 'example3'

So i suppose it is must  to run crypt tool for the binary created using your functions.
Pls clarify

' ' ' ' 
This line is not clear:

It is important to know that your program if it is compiled, is NOT obfuscated. You first have to execute the crypt tool on it.
' ' ' '

**Comment by kutoga** [comment]
You are right: Just after the compilation process, the resulting binary is **not** yet obfuscated.

Actually, this has to be done once by executing the `crypt`-tool.

One might also think about a resulting program that re-applies the obfuscation process to itself. This could be done every time the program is executed.
